codegen/src/module.rs

```diff
@@ -140,12 +140,13 @@ impl Parse for Module {
                         ref expr,
                         ident,
                         attrs,
+                        ty,
```

Comment:
```suggestion
                        ref ty,
```

Reply:
I'll fix `ExportedConst` not to take a `Box`.